src/test/java/Homework21.java

```diff
@@ -0,0 +1,21 @@
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
+
+public class Homework21 extends BaseTest{
+    @Test (description = "renamePlaylist")
+
+    public void renamePlaylist () throws InterruptedException{
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+
+        loginPage.logInWithRightCredentials();
+        homePage.doubleClickPlaylist()
+                .enterNewPlaylistName();
```

Comment:
Please, add Assert

**One test - One Assert**

For example

![Clean_Test](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/104761482/5001a36e-9f72-461f-933b-69720780d86c)